Task2/src/middlewares/middlewares.js

```diff
@@ -1,6 +1,9 @@
-exports.logErrors = (err, req, res, next) => {
-    console.error(err.stack);
-    next(err);
+const winston = require('../logger');
+
+exports.handleError = (err, req, res, next) => {
+    winston.logger.error(err.stack);
+    res.status(500).send('Internal server error');
+    next();
```

Comment:
There is no need to add `next()` call after response sending. It's not an error, but just unreachable code.